src/config.ml

```diff
@@ -202,6 +216,7 @@ module Conf_map = struct
           * [ `Udp | `Tcp of [ `Server | `Client ] option ])
           k
       (* see socket.c:static const struct proto_names proto_names[] *)
+    | Protocol_flags : Protocol_flag.t list k
```

Comment:
I considered using a set, but a list is simpler and there's at most half a dozen different flags at the moment.